api/jobs.go

```diff
@@ -151,6 +143,16 @@ func (api *API) GetJobsHandler(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (api *API) setUpPagination(w http.ResponseWriter, offsetParam string, limitParam string, ctx context.Context) (int, int, error) {
+	paginator := pagination.NewPaginator(api.cfg.DefaultLimit, api.cfg.DefaultOffset, api.cfg.DefaultMaxLimit)
+	offset, limit, err := paginator.ValidatePaginationParameters(offsetParam, limitParam)
+	if err != nil {
+		log.Error(ctx, "pagination validation failed", err)
+		return 0, 0, err
+	}
+	return offset, limit, err
```

Comment:
Maybe consider letting the client log the error however they want? 
It would simplify the code a bit and seems to be consistent with how we deal with these errors (`log.Error` followed by `http.Error`). 
We wouldn't need to pass the context as a parameter and simply do: 
```suggestion
	return paginator.ValidatePaginationParameters(offsetParam, limitParam)
```

Reply:
yes, I was unsure where to put the logging but I like your suggestion of simplifying the code - done